neofetch

```diff
@@ -8823,6 +8833,82 @@ EOF
     fi
 }
 
+ssh_connect(){
+    ssh_command="ssh"
+    ssh_opts="-t"
```

Comment:
This should be `ssh_opts=(-t)` as this opens up the possibility to make the flags configurable.